readme.md

```diff
@@ -1,88 +1,197 @@
+```js
+{
+  "requestId": "HFMlljrzKEiZmhUNDx7Z",
+  "visitorId": "kHqPGWS1Mj18sZFsP8Wl",
+  "visitorFound": true,
+  "incognito": false,
+  "bot": { "probability": 0.96 },
+  "browserName": "Chrome",
+  "browserVersion": "85.0.4183",
+  "os": "Mac OS X",
+  "osVersion": "10.15.6",
+  "device": "Other",
+  "ip": "192.65.67.131",
+  "ipLocation": {
+    "accuracyRadius": 100,
+    "latitude": 37.409657,
+    "longitude": -121.965467
+    // ...
+  }
+}
+```
 
-The documented JS API follows [Semantic Versioning](https://semver.org).
-Using undocumented features is at you own risk.
+🍿 [Live demo](https://fingerprintjs.com/demo)
 
-## Contribution
+⏱ [How to upgrade from Open Source to Pro in 30 seconds]()
 
-### Development playground
+📕 [FingerprintJS Pro documentation](https://dev.fingerprintjs.com)
 
-```bash
-# Make sure you have Yarn installed
-yarn install
-yarn playground:start # Add '--port 8765' to change the server port
-```
+## Open-source version reference
 
-Then open http://localhost:8080 in a browser.
-It's reloaded every time the source code is changed.
+### Installation
 
-To build the playground distribution code (e.g. to upload to a static server), run:
+The library is shipped in various formats:
 
-```bash
-yarn playground:build
-```
+- Global variable
+    ```html
+    <script
+      async src="https://cdn.jsdelivr.net/npm/@fingerprintjs/fingerprintjs@3/dist/fp.min.js"
+      onload="/* FingerprintJS... */"
+    ></script>
+    ```
+- UMD
+    ```js
+    require(['https://cdn.jsdelivr.net/npm/@fingerprintjs/fingerprintjs@3/dist/fp.umd.min.js'], (FingerprintJS) => {/* ... */});
+    ```
+- EcmaScript module
+    ```js
+    import FingerprintJS from '@fingerprintjs/fingerprintjs';
+    ```
+- CommonJS
+    ```js
+    const FingerprintJS = require('@fingerprintjs/fingerprintjs');
+    ```
+
+### API
+
+- `FingerprintJS.load({ delayFallback?: number }): Promise<Agent>`
+
+    Builds an instance of Agent and waits a delay required for a proper operation.
+    `delayFallback` is an optional parameter that sets duration (milliseconds) of the fallback for browsers that don't support [requestIdleCallback](https://developer.mozilla.org/en-US/docs/Web/API/Window/requestIdleCallback);
+    it has a good default which isn't recommended to change.
+
+- `agent.get({ debug?: boolean }): Promise<{ visitorId: string, components: {/* ... */} }>`
+
+    Gets the visitor identifier.
+    `debug: true` prints debug messages to the console.
+    `visitorId` is the visitor identifier.
+    `components` is a dictionary of components that have formed the identifier;
+    each value is an object like `{ value: any, duration: number }` in case of success
+    and `{ error: object, duration: number }` in case of an unexpected error during getting the component.
+
+- `FingerprintJS.hashComponents(components: object): string`
+
+    Converts a dictionary of components (described above) to a short hash string a.k.a. a visitor identifier.
```

Comment:
.. into a short hash string

Reply:
Do you mean to leave or discard the "a.k.a. ..." part?